README.md

```diff
@@ -45,13 +59,37 @@ yarn scripts:addTrip
 
 You can run a migration service to add aggregation data on your timeseries. This is necessary because the trips documents can be huge and negatively impact the app performances. Therefore, we rely on an aggregated trip view on the app side. If the aggregation is missing, the trip won't be displayed.
 
+⚠️ Make sure to update your development URL in `./konnector-dev-config.json` in the field `"COZY_URL"` if you do not wish to use the default URL `http://cozy.localhost:8080`.
+
+⚠️ Your instance database must contain in its `io.cozy.accounts` the following document as-is in order to be able to find the `geoseries` collections, please insert it manually or with a mango request into the associated database:
```

Comment:
This is not necessary if you run `yarn fixtures`, that already does that

Reply:
From my test this morning, `yarn fixtures --url` did not create an account. I had to add it, else CoachCO2 would fail to fetch the collections.

Reply:
je pense c'est parce que le yarn fixtures fait un yarn import && yarn scripts et que le -u ou --url ne fonctionne que sur la derniere commande, à savoir yarn script. En gros il doit faire yarn import && yarn script -u mais pas yarn import -u && yarn script -u